app/views/surveys/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<div>
+  <div >
+    <%= form_for @survey, id: "form" do |f| %>
+      <label> Name of Survey: </label>
+      <%= f.text_field :name %>
+      <label> Description of Survey: </label>
+      <%= f.text_field :description %></br>
+      <label> Category of Survey: </label>
+      <%= f.text_field :category %></br>
+      <label> Type of Survey: </label>
+      <%= f.select :survey_type , raw("<option>Private</option><option>Public</option>")%>  
```

Comment:
can we use html_safe/sanitize here?
please read about html_safe, sanitize and raw